src/internal/WixInternal.TestSupport/Builder.cs

```diff
@@ -46,10 +57,13 @@ public string[] BuildAndQuery(Action<string[]> buildFunc, bool validate, params
                     "-intermediateFolder", intermediateFolder,
                 };
 
-                if (this.ExtensionType != null)
+                if (null != this.ExtensionType)
```

Comment:
Yoda comparisons aren't needed in managed code. (They're still officially style for WiX C code, however.)

Reply:
thanks for the tip